.github/workflows/release.yml

```diff
@@ -8,9 +8,6 @@ jobs:
     if: "!contains(github.event.head_commit.message, 'ci skip') && !contains(github.event.head_commit.message, 'skip ci')"
     steps:
       - uses: actions/checkout@v2
-        with:
-          # Ensure that git uses your token with write access to the repo
-          token: ${{ secrets.GH_TOKEN }}
```

Comment:
not needed?

Reply:
that's right, the checkout action detects the token automatically